apps/frontend/src/app/1_atoms/DappLocked/DappLocked.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+
+import { Trans } from 'react-i18next';
+
+import { SovrynLogo } from '../../2_molecules';
+import sovrynLogo from '../../../assets/images/sovryn-small-logo.svg';
+import { translations } from '../../../locales/i18n';
+
+export const DappLocked: FC = () => (
+  <div className="flex-grow bg-black flex items-center justify-center">
+    <div className="sm:rounded sm:w-96 sm:px-14 sm:py-10 sm:bg-gray-90">
+      <SovrynLogo
+        image={sovrynLogo}
+        className="w-8 mx-auto mb-5"
+        dataAttribute="header-logo"
+      />
+      <p className="mb-0 max-w-xs mx-auto text-center">
```

Comment:
Add required spacing according to the design and change `font-size` (should be `0.875rem` instead of `1rem`).

Design:
<img width="254" alt="image" src="https://user-images.githubusercontent.com/14878713/220080236-c4bde7a6-2093-47e9-b4c0-ee0c73bce18b.png">

Implementation:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/14878713/220080332-ccf7c024-dcd5-4f35-bc13-988a38ade547.png">